### PR TITLE
added Three sum file (issue #4140)

### PR DESCRIPTION
**Pull request by anshmishra010**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

We have to find the sum of the target value using using three digit present in the list.
I'll provide the optimized solution using two pointer technique.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#4140

### CODE REVIEW

**Comment by gat786** [comment]
The readme seems to be not updated. Push the commit in which we updated the readme.

**Comment by anshmishra010** [comment]
done

File: C-Plus-Plus/cp/Three_Sum.cpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+     Given an array and a value, find if there exists three
+     number whose sum is equal to the given value.
+     If found then output will be true else false.
+
+*/
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main()
```
Comment (HarshCasper) [inline C-Plus-Plus/cp/Three_Sum.cpp:12]: Push all the main code logic in a separate function and use main() for driver code

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — COMMENTED** [review]
@anshmishra010 Do the changes mentioned.
Rest looks good to merge.

**Comment by anshmishra010** [comment]
Actually the thing is I have deleted the branch from which I have pushed.

**Comment by ankitaggarwal23** [comment]
@anshmishra010 Okay I'm closing this PR. Open a new one and do the changes, we will review

**Pull request closed without merging**